### PR TITLE
dashboard: fix a render issue

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

Fix a render issue:

7eb8eb9e-32fd-491a-9d7c-31bf6bc8eb12

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by hfxsd — APPROVED** [review]

File: dashboard/dashboard-resource-manager.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ summary: 介绍如何使用 TiDB Dashboard 的资源管控页面查看资源管
     - `oltp_read_write`：数据读写平衡的负载，根据类似 `sysbench oltp_read_write` 的负载模型预测。
     - `oltp_read_only`：数据读取较重的负载，根据类似 `sysbench oltp_read_only` 的负载模型预测。
 
-    ![基于硬件部署估算容量](/media/dashboard/dashboard-resource-manager-calibrate-by-hardware.png)
+  ![基于硬件部署估算容量](/media/dashboard/dashboard-resource-manager-calibrate-by-hardware.png)
 
     用户资源分组总请求单元 (Total RU of user resource groups) 表示当前除 `default` 用户外的 RU 总量。当该数值小于容量估算值时，系统会发出提醒。系统预定义的 `default` 资源组默认拥有无限用量。当所有用户都属于 `default` 资源组时，资源分配方式与关闭资源管控时相同。
```
Comment (HuSharp) [inline dashboard/dashboard-resource-manager.md:50]: 这一段需要对齐吗？

**Review by HuSharp — APPROVED** [review]

File: system-variables.md [hunk 2]
```diff
@@ -142,7 +142,7 @@ SET  GLOBAL tidb_distsql_scan_concurrency = 10;
 - 是否持久化到集群：是
 - 类型：枚举型
 - 默认值：`SIMPLE`
-- 可选值：`SIMPLE`.
+- 可选值：`SIMPLE`
```
Comment (Oreoxmt) [inline system-variables.md:145]: @ran-huang PTAL

Comment (ran-huang) [reply system-variables.md:145]: LGTM

**Pull request merged**